src/gtest/test_libzendoo.cpp

```diff
@@ -126,20 +150,20 @@ TEST(ZendooLib, TestProof)
 
     //Inputs
     unsigned char end_epoch_mc_b_hash[32] = {
-        78, 85, 161, 67, 167, 192, 185, 56, 133, 49, 134, 253, 133, 165, 182, 80, 152, 93, 203, 77, 165, 13, 67, 0, 64,
-        200, 185, 46, 93, 135, 238, 70
+        157, 219, 85, 159, 75, 56, 146, 21, 107, 239, 76, 31, 208, 213, 230, 24, 44, 74, 250, 66, 71, 23, 106, 4, 138,
+        157, 28, 43, 158, 39, 152, 91
     };
 
     unsigned char prev_end_epoch_mc_b_hash[32] = {
```

Comment:
It would be better to avoid hardcoding the value 32 (similarly to what has been done with SC_FIELD_SIZE)